### ISSUE TITLE
Using with DDG Final does nothing

### ISSUE THREAD

**Issue by jaoxford** [hunterbridges/DDGModernizer#6]
Hey, thanks for the great work on putting this tool together. However it doesn't work for me with DDG Final!

I got this with the english translation patch (bundled in at version 1.0.2.)

I've set the EXE path & Game Folder path (they are the same as the defaults), and I have set my resolution to 1080p. However when I open up the game it is still running in 480p, and not in borderless window mode either.

I have also tried to run this using administrator. But alas, no change.

Here's how the moderniser looks from my end:
![image](https://github.com/hunterbridges/DDGModernizer/assets/26263928/12fbc8f0-c845-437a-b816-3e8c268798ed)

**Comment by Ligutrino** [comment]
Yes, borderless mode don't work here too. About the resolution you have to use ddgfncal.exe Advanced options.

**Comment by jaoxford** [comment]
> Yes, borderless mode don't work here too. About the resolution you have to use ddgfncal.exe Advanced options.

That's what I did in the end. Disappointingly the modernizer had no affect at all.
